mode-apis-base/src/alertingModels.ts

```diff
@@ -163,12 +163,47 @@ export const isMetricsThresholdAlertCondition = (obj: unknown): obj is MetricsTh
 
 
 
-/**
- * TODO - Define the attributes required for AlertAction
- */
-export interface AlertAction {
+export interface AlertMessageTemplate {
+    readonly title: string;
+    readonly message: string;
+    readonly recoveryMessage?: string;
+}
+
+export type AlertRecipientType = 'email';
+
+export interface AlertRecipient {
+    recipientType: AlertRecipientType;
 }
 
+export interface AlertRecipientEmail extends AlertRecipient {
+    readonly recipientType: 'email';
+    readonly email: string;
```

Comment:
Do we want to allow sending emails to multiple recipients?
If we don't FOR NOW but we think we might support it in the future, how about we define it as an Array of strings so that if we allow multiple emails in the future, we don't need to change the schema? The front end can ask the user to enter only 1 email but will store it in an Array.

```suggestion
    readonly emails: readonly string[];
```

Reply:
> Do we want to allow sending emails to multiple recipients?

Yes, I think so.

But, in my design, we can achieve that by AlertAction definition.

```
{
  filter: {...},
  recipients: [
    {
        recipientType: AlertRecipientType.EMAIL,
        email: "bananaumai@tinkermode.com"
    },
    {
        recipientType: AlertRecipientType.EMAIL,
        email: "bill@tinkermode.com"
    }
  ]
}
```

Reply:
Ok, this works.